fun.vnote.app.VNote.yaml

```diff
@@ -0,0 +1,65 @@
+id: fun.vnote.app.VNote
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-23.08'
+command: vnote
+finish-args:
+  - --share=ipc
+  # checking latest update information
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=xdg-documents
+  # reading host Git's configs
+  - --filesystem=~/.gitconfig
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess   
+rename-icon: vnote
+rename-desktop-file: vnote.desktop
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh  
+modules:
+  - name: vnote
+    buildsystem: qmake
+    config-opts:
+      - PREFIX = ${FLATPAK_DEST}
+      - LIBS+=-L/app/lib
+    build-options:
+      env:
+        QMAKEPATH: /app/lib
+    sources:
+      - type: git
+        url: https://github.com/vnotex/vnote.git
+        tag: v3.17.0
+        commit: 4ab2033a81ee43e2a4b836b235153e20456fffea
+    post-install:
+      - desktop-file-edit --set-icon ${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/vnote.desktop
+
+  - name: metainfo
+    buildsystem: simple
+    build-commands:
+      - install -p -D -m 0644 ${FLATPAK_ID}.metainfo.xml -t "${FLATPAK_DEST}/share/appdata/";
+    sources:
+      - type: file
+        path: fun.vnote.app.VNote.metainfo.xml
+
```

Comment:
Add this to the vnote module, add a second source "type: file" for the appdata, install it in post-install or build-commands